redis/client_test.go

```diff
@@ -67,6 +68,11 @@ func TestParse(t *T) {
 	assert.Equal(t, ErrorReply, r.Type)
 	assert.Equal(t, "ERR unknown command 'foobar'", r.Err.Error())
 
+	// wrongtype reply
+	r = parseString("-WRONGTYPE Operation against a key holding the wrong kind of value\r\n")
+	assert.Equal(t, ErrorReply, r.Type)
+	assert.Equal(t, "WRONGTYPE Operation against a key holding the wrong kind of value", r.Err.Error())
+
```

Comment:
I just wanted to make sure it doesn't break our driver, since radix doesn't mention anything about that it can handle 3.0. I went through release log from 2.2 to 3.0 and this is the only change they made that will affect us.